rest/model/task_queue.go

```diff
@@ -13,6 +13,7 @@ type APITaskQueueItem struct {
 	Requester           *string     `json:"requester"`
 	Revision            *string     `json:"gitspec"`
 	Project             *string     `json:"project"`
+	ProjectIdentifier   *string     `json:"project_identifier"`
```

Comment:
is omit_empty something that would work if we put this on here? I don't know if we return taskQueue items in any rest routes, but users are going to be confused I think by the fact that this is empty, if it is returned